installers/linux-shared/after-upgrade.sh.erb

```diff
@@ -0,0 +1,29 @@
+<%
+#########################################################################
+# Copyright 2016 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########################################################################
+-%>
+
+<% if type == 'rpm' -%>
+    # do nothing
```

Comment:
This if check is not required. It improves readability. But can't we add a comment instead to show that if the type is rpm, we do nothing?